CodeScript/mains.js

```diff
@@ -65,7 +64,7 @@ function initializeBookStore() {
       document.querySelector('#book-list').addEventListener('click', (e) => {
         if (e.target.classList.contains('delete')) {
           const row = e.target.parentElement.parentElement;
-          const title = row.querySelector('td:first-child').textContent;
+          const title = row.querySelector('#name').textContent;
           this.removeBook(title);
```

Comment:
- [ ] I cannot highlight the remove book method itself, so I am highlighting this as a point of reference. When I add two books of the same title and delete one, the two books are removed. Kindly fix that by implementing the delete feature with a unique identifier, like generated number or date, instead of using the title.